features/step_definitions/api/services/service_plans_steps.rb

```diff
@@ -106,3 +106,18 @@
   visit admin_service_service_plans_path(default_service)
   @plan.destroy
 end
+
+When "there are other services with service plans" do
+  FactoryBot.create_list(:service, 3, account: @provider).each do |service|
+    assert_equal 1, service.service_plans.count
+  end
+end
+
+Then "only service plans of the current service are listed" do
+  FactoryBot.create(:service_plan, issuer: default_service, name: 'An extra plan')
+  FactoryBot.create(:service_plan, issuer: default_service, name: 'Another extra plan')
+
+  @plans = default_service.service_plans
+  visit admin_service_service_plans_path(default_service)
+  assert_plans_table @plans
+end
```

Comment:
Yeah, OK, good.

The only thing is that I wanted to add some extra service plans, to make sure they are all visible, not only the default one, because if it's just "Default" - it could be coming from another service, as this is the default name of the service plan.